docs/tokens.md

```diff
@@ -3,137 +3,239 @@ id: tokens
+
+### Standards
 
 Even though the concept of a token is simple, they have a variety of complexities in the implementation. Because everything in Ethereum is just a smart contract, and there are no rules about what smart contracts have to do, the community has developed a variety of **standards** (called EIPs or ERCs) for documenting how a contract can interoperate with other contracts.
 
-You've probably heard of the **ERC20** standard, and that's why you're here.
+You've probably heard of the [ERC20](#erc20) or [ERC721](#erc721) token standards, and that's why you're here.
+
+---
 
 ## ERC20
 
-An ERC20 token is a contract that keeps track of a `mapping(address => uint256)` that represents a user's balance. These tokens are _fungible_ in that any one token is exactly equal to any other token; no tokens have special rights or behavior associated with them. This makes ERC20 useful for things like a medium of exchange currency, general voting rights, staking, and more.
+An ERC20 token contract keeps track of [_fungible_ tokens](#different-kinds-of-tokens): any one token is exactly equal to any other token; no tokens have special rights or behavior associated with them. This makes ERC20 tokens useful for things like a **medium of exchange currency**, **voting rights**, **staking**, and more.
+
+OpenZeppelin provides many ERC20-related contracts. On the [`API reference`](api/token/ERC20) you'll find detailed information on their properties and usage.
+
+### Constructing an ERC20 Token Contract
+
+Using OpenZeppelin, we can easily create our own ERC20 token contract, which will be used to track _Gold_ (GLD), an internal currency in a hypothetical game.
+
+Here's what our GLD token might look like.
+
+```solidity
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
+
+contract GLDToken is ERC20, ERC20Detailed {
+    constructor(uint256 initialSupply) ERC20Detailed("Gold", "GLD", 18) public {
+        _mint(msg.sender, initialSupply);
+    }
+}
+```
+
+OpenZeppelin contracts are often used via [inheritance](https://solidity.readthedocs.io/en/latest/contracts.html#inheritance), and here we're reusing the [`ERC20`](api/token/ERC20#erc20) (for the basic standard implementation) and [`ERC20Detailed`](api/token/ERC20#erc20detailed) (to get the [`name`](api/token/ERC20#ERC20Detailed.name()), [`symbol`](api/token/ERC20#ERC20Detailed.symbol()) and [`decimals`](api/token/ERC20#ERC20Detailed.decimals()) properties) contracts. Additionally, we're creating an `initialSupply` of tokens, which will be assigned to the address that deploys the contract.
+
+_For a more complete discussion of ERC20 supply mechanisms, see [our advanced guide](https://forum.zeppelin.solutions/t/how-to-implement-erc20-supply-mechanisms/226/)_.
```

Comment:
This can link to the guide on the docs now.

Reply:
Good catch